evaluations/list_detail_mvc_provider/lib/src/users/user_list_controller.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:list_detail_mvc_provider/src/users/user.dart';
+import 'package:list_detail_mvc_provider/src/users/user_repository.dart';
+
+/// This is the controller (the C in MVC). It acts as an intermediary between
```

Comment:
How are you feeling about MVC after writing up this kind of structure? Does the explanation feel natural and in line with the code you normally write and see from the community?

Reply:
I agree with what you said earlier: I think the MVC pattern is used everywhere, although the terminology is largely ignored and therefore the M+V+C parts are not as precisely defined. I'm personally fine with it being a bit more freestyle, but I think the idea to stake out some terminological baseline is great, and we should pursue it.

I think the following is a really useful framework (not sure if it's pure MVC or something adjacent):

* `M` is data
* `V` is widgets
* `C` is the binding (basically a facade for the `V` to work with the `M`)
* `Repository` is an abstraction of the access & synchronization of `M`
* `Feature` is a user-centric partition of the app

Reply:
Sounds good -- As you said, we don't want to teach everything or even much, but I'd be curious to chat with the community a bit more about this MVC structure and see if it hits that sweet spot from the requirements doc: "...the new template SHOULD NOT be overwhelming to new Flutter developers, and it MUST NOT be overwhelming to experienced Flutter developers. Experienced Flutter developers should generally feel comfortable with the way it's built"